wgengine/capture/capture.go

```diff
@@ -65,6 +65,9 @@ const (
 	// SynthesizedToPeer indicates the packet was generated from within tailscaled,
 	// and is being routed to a remote Wireguard peer.
 	SynthesizedToPeer Path = 3
+
+	// PathDisco indicates the packet is metadata about a valid disco frame.
```

Comment:
drop "valid"? invites more questions. (like: is there a path for invalid disco frames?)

And is it just "metadata"? It also contains the payload. Maybe you want something like "a decrypted disco message"?